apps/modernization-ui/src/components/Table/style.scss

```diff
@@ -84,3 +84,7 @@ table.usa-table thead tr {
     border-bottom-width: 2px;
     border-bottom-style: solid;
 }
+
+table.usa-table .head-name {
```

Comment:
issue:  This may not be enough to enforce the `14px` on all of the tables.  There are two table implementations `Table` and `SortableTable` with the latter delegating `td` creation to the caller (issue for another day).  Click the events or demographics tab to see tables that still have `16px` font.

I'm not sure that `.head-name` is needed here, should be able to select all `th` or `td` under `table.use-table`.  [This](https://github.com/CDCgov/NEDSS-Modernization/blob/f3caa8f7a8e9e7c206c507c64e8a7a7fd1b1c8ce/apps/modernization-ui/src/components/Table/style.scss#L17) is also going to have to be removed so it doesn't override the specified styling.